banner.html

```diff
@@ -11,7 +11,7 @@ <h1>Health-RI RDF Validator</h1>
                 <p> This service allows you to validate RDF content against <a href="https://www.w3.org/TR/shacl/">SHACL
                         shapes</a>.
                     The shapes conform to the <a
-                        href="https://health-ri.atlassian.net/wiki/spaces/FSD/pages/121110529/Core+Metadata+Schema+Specification">Health-RI
+                        href=https://github.com/Health-RI/health-ri-metadata/>Health-RI
```

Comment:
Could we link this to a `latest` release? Also depends on what we end up putting in the description of the release.

Reply:
will delete this banner file. It not used (i figured out when updating the link) The banner is located in the config. file.